code/modules/admin/admin_verbs.dm

```diff
@@ -637,6 +639,30 @@ var/list/admin_verbs_mod = list(
 	message_admins("<span class='notice'>[ckey] creating an admin explosion at [epicenter.loc].</span>")
 	feedback_add_details("admin_verb","DB") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/drop_emp()
+	set category = "Special Verbs"
+	set name = "Drop EMP"
+	set desc = "Cause an EMP of varying strength at your location."
+

```

Comment:
You can attach a singuloth to yourself with atom locking already, though said person will probably be eaten 2 seconds later.